splunk_add_on_ucc_framework/__init__.py

```diff
@@ -809,14 +823,14 @@ def _generate(source, config, ta_version, outputdir=None):
         app_config.add_section('ui')
 
     app_config['launcher']['version'] = ta_version
-    app_config['launcher']['description'] = manifest['info']['description']
+    app_config['launcher']['description'] = manifest.get_description()
 
     app_config['id']['version'] = ta_version
 
     app_config['install']['build'] = str(int(time.time()))
-    app_config['package']['id'] = manifest['info']['id']['name']
+    app_config['package']['id'] = ta_name
```

Comment:
Should we keep an assert statement to confirm both `manifest['info']['id']['name']` and ta_name are same?

Reply:
Let's create a test for this one and if someone changes the behaviour then test will test us.